src/init.cpp

```diff
@@ -64,6 +65,10 @@ using namespace std;
 CWallet* pwalletMain = NULL;
 #endif
 bool fFeeEstimatesInitialized = false;
+static const bool DEFAULT_PROXYRANDOMIZE = true;
+static const bool DEFAULT_REST_ENABLE = false;
+static const bool DEFAULT_SAFEMODE = true;
```

Comment:
I'd prefer to use `DEFAULT_DISABLESAFEMODE = false` then use that instead of negating this everywhere it is used.
(agree that  in retrospect the option should have been called safemode and acted in this way, but there's no way back now, and having this consistent makes it easier to link it to the constant)